phinx.config.php.dist

```diff
@@ -0,0 +1,38 @@
+<?php
+
+// copy this file to phinx.config.php, or your preferred name/location
+
+// keep this here to load the dotenv package
+// only if the config file is used
+$Loader = (new josegonzalez\Dotenv\Loader(__DIR__ . '/.env'))
+    ->parse()
+    ->putenv();
+
+// return the configuration
+return [
+    'paths' => [
+        'migrations' => __DIR__ . '/migrations',
+    ],
+    'environments' => [
+        'default_migration_table' => 'phinxlog',
+        'default_database' => getenv('APP_ENV'),
+        'develop' => [
```

Comment:
I think this should match the `APP_ENV` value in .env.example - "development"

Reply:
Good catch. Thanks!